testpmd/hooks/pre-run.yml

```diff
@@ -43,6 +66,14 @@
   include_tasks: config/sriov.yaml
   when: enable_sriov|default(true)|bool
 
+# Need to check with S why this is required
+- name: Apply label to worker for cnf app
+  shell: |
+    export KUBECONFIG={{ kubeconfig_path }}
+    {{ oc_tool_path }} label node worker-0 examplecnf.openshift.io/trex=""
```

Comment:
The problem with this task is that the worker node name has to be constant. I see most of the times worker-0 node is not deployed successfully. The node-mange-operator which is invoked at https://github.com/dallas-telco-lab/example-cnf-config/blob/master/testpmd/hooks/install.yml#L19 will apply the required labels. Was it not working?